error-messages/user.messages.js

```diff
@@ -5,6 +5,9 @@ const WRONG_CREDENTIALS = 'WRONG_CREDENTIALS';
 const USER_NOT_AUTHORIZED = 'USER_NOT_AUTHORIZED';
 const EMAIL_ERROR = 'EMAIL_ERROR';
 const INVALID_PERMISSIONS = 'INVALID_PERMISSIONS';
+const PASSWORD_RECOVERY_ATTEMPTS_LIMIT_EXCEEDED =  'PASSWORD_RECOVERY_ATTEMPTS_LIMIT_EXCEEDED';
+const TOKEN_NOT_VALID = 'TOKEN_NOT_VALID';
```

Comment:
I think it is better to make this message more specific and describe which token is not valid